### ISSUE TITLE
to_class_case implementation different than from what Homebrew expects

### ISSUE THREAD

**Issue by jennydaman** [axodotdev/cargo-dist#551]
I am trying to use `cargo-dist` for my program called [mni2mz3](https://github.com/FNNDSC/mni2mz3) however the homebrew installer is not working.

https://github.com/axodotdev/cargo-dist/blob/c10eb4625374f0c2b673d6ad3961f3a851e6b77d/cargo-dist/src/tasks.rs#L1419

The return of `to_class_case` is passed to the template here:

https://github.com/axodotdev/cargo-dist/blob/c10eb4625374f0c2b673d6ad3961f3a851e6b77d/cargo-dist/templates/installer/homebrew.rb.j2#L1

For the project name `mni2mz3`, the generated `formula_class` is `Mni2Mz3` however this does not work with homebrew (version 4.1.19)

```shell
$ brew install FNNDSC/homebrew-mni2mz3/mni2mz3
# -- snip --
Error: No available formula with the name "fnndsc/mni2mz3/mni2mz3".
In formula file: /usr/local/Homebrew/Library/Taps/fnndsc/homebrew-mni2mz3/Formula/mni2mz3.rb
Expected to find class Mni2mz3, but only found: Mni2Mz3.
```

**Comment by mistydemeo** [comment]
Thanks for the report; I'll look into getting this fixed.

**Comment by mistydemeo** [comment]
I have a fix that should make our next bugfix release. I've used your app's name as a test case to ensure it'll work.

**Comment by mistydemeo** [comment]
This should be fixed in the new 0.4.3 release. Thanks again for the report!
